src/common/svg/PlusIcon.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
Not a huge deal, but this SVG has unnecessary meta info in it. If you run it through a svg minifier, these will be removed.